content/rest-api/resources.md

```diff
@@ -1079,3 +1081,192 @@ Endpoints for the published products are only available starting the 2.0 version
 
 ::: panel-link Want more details about the published product resource? [Check its endpoints here!](/api-reference.html#Publishedproducts)
 :::
+
+## Reference entities
+
+### Reference entity record (3.x and EE only)
+
+Reference entities have been introduced in the PIM in the 3.0 version.
+
+//TODO Def!
```

Comment:
ok, but we have to not forget it!

Reply:
I wrote it in the description of this PR, the things that are left to do.

Reply:
@Delphineray Can you help us here? :)

Reply:
Done!